### ISSUE TITLE
CDS missing from ConvertDb-genbank output

### ISSUE THREAD

**Issue by ssnn-airr** [immcantation/changeo#159]
**[Original report](https://bitbucket.org/kleinstein/changeo/issues/159) by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

Junction CDS seems is missing from the ConvertDb-genbank output in v0.4.6 and current tip. It is not missing in v0.4.5. Tested with example S43 data.

**Comment by ssnn-airr** [comment]
**Original comment by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

@{557058:17032cad-7aa5-47e3-a2ff-083ece1e1478} , I think this is fixed in 618970c. Could you test, please?

My test command was:

```
ConvertDb.py genbank -d S43_update.tab
--product "immunoglobulin heavy chain"
--db "IMGT/GENE-DB"
--inf "IgBLAST:1.7.0"
--organism "Homo sapiens"
--sex Male
--tissue "Peripheral blood"
--cell-type "B cell"
--outdir output_asn
--isolate S43
--cf CPRIMER
--nf DUPCOUNT
--asis-id
--allow-stop
--format changeo
--asn
--sbt S43_template.sbt
```

‌

**Comment by ssnn-airr** [comment]
**Original comment by Julian Zhou (Bitbucket: [jqz](https://bitbucket.org/jqz), GitHub: [julianqz](https://github.com/julianqz)).**

----------------------------------------

Yes, I tested on my data and the CDS/junction feature is back. Thumbs up!

**Comment by ssnn-airr** [comment]
**Original comment by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

Fixed in 618970c.
